keylime/tpm1.py

```diff
@@ -813,6 +813,20 @@ def check_quote(self,agent_id,nonce,data,quote,aikFromRegistrar,tpm_policy={},im
 
         return self.check_pcrs(agent_id,tpm_policy,pcrs,data,False,ima_measurement_list,ima_whitelist)
 
+    def sim_extend(self,hashval_1,hashval_0=None,hash_alg=None):
+        # simulate extending a PCR value by performing TPM-specific extend procedure
+
+        if hash_alg is None:
```

Comment:
i think having hash_alg here is a bit confusing since it will just always use sha1.  overall, hash_alg isn't even used in tpm2.  i recommend removing this param entirely.

Reply:
Ah, good eye!  I think the logic is already handled in `self.hashdigest` for tpm2, so we don't actually need to know the algorithm in `sim_extend`.  

I'll go ahead and fix this up.